.travis.yml

```diff
@@ -7,9 +7,9 @@ env:
   matrix:
     - TEST=javascript
     # need node installs for app_manager.XpathValidatorTest
-    - TEST=python NOSE_DIVIDED_WE_RUN=05 JS_SETUP=yes
-    - TEST=python NOSE_DIVIDED_WE_RUN=6a JS_SETUP=yes
-    - TEST=python NOSE_DIVIDED_WE_RUN=bf JS_SETUP=yes
+    - TEST=python NOSE_DIVIDED_WE_RUN=03 JS_SETUP=yes
+    - TEST=python NOSE_DIVIDED_WE_RUN=69 JS_SETUP=yes
+    - TEST=python NOSE_DIVIDED_WE_RUN=ba JS_SETUP=yes
```

Comment:
these should be `03` `4a` `bf` - each string represents a range, and they should be contiguous or it will skip some tests. As is, you'd be skipping `46` (that is, any test whose "name" MD5 starts with `4`, `5`, or `6`) and probably `af`. I think you might get an error for `ba` since `b` > `a` so not a valid range.

As for how to pick the values, I'd probably go with `04` `5a` `bf` and see if that works. If the first is still taking to long, then go to `03` `4a` `bf`. I'm assuming because of the change you made here, the first runner is hitting timeout, the second has time to spare, and the third is about right?